requirements.txt

```diff
@@ -1,8 +1,7 @@
-ordereddict==1.1
-argparse==1.2.1
-python-dateutil==2.2
+argparse
+ordereddict
+python-dateutil
 matplotlib==1.3.1
-nose>=1.3.0
 numpy==1.8.0
-pymongo>=3.3.0
-psutil>=2.0
+pymongo
+psutil
```

Comment:
Let's leave the pinning of the dependencies in the `requirements.txt` for now. We should open another issue/PR to clean up dependencies to reflect current support of Python 2.7 and 3.x.